internal/start/start.go

```diff
@@ -827,7 +859,7 @@ EOF
 	}
 
 	// Start Storage.
-	if utils.Config.Storage.Enabled && !isContainerExcluded(utils.Config.Storage.Image, excluded) {
```

Comment:
Why does moving the same check earlier make a difference?

Reply:
Good catch, I think the second part of the condition is unnecessary I can just check over the `isStorageEnabled`.

As for the why I moved it to a dedicated value it's to ensure that we always check the same values when we check "isStorageEnabled" aka: "is the config true, and storage not excluded from the start".

I had this error where, if you had storage `enabled` in the config but excluded with the flags, then, the storage container was not started (as expected) but the "imageTransform" container was attempting to start (because it was only checking over the config value which was true) and not against the excluded flags. But since the Image container depends on the storage to work properly, the stack was failing to start.